defaults/main.yml

```diff
@@ -114,3 +114,13 @@ kops_default_kubeControllerManager:
     - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
 kops_default_kubeScheduler:
   enableProfiling: false
+
+###
+### Verify dockerconfig via get ratelimited
+### https://docs.docker.com/docker-hub/download-rate-limit/#how-can-i-check-my-current-rate
+###
+kops_default_dockerconfig_creds:
```

Comment:
This should also be added to the Readme in: https://github.com/Flaconi/ansible-role-kops#additional-variables

**Q:** what happens when somebody does not want to specify a docker configuration and want to continue to use anonymous auth?

Reply:
> Question:

They just don't defined `kops_dockerconfig_creds` this map, then they will go with anonymous user.
`kops_default_dockerconfig_creds`: I will update it in readme